python/ray/serve/_private/replica.py

```diff
@@ -508,19 +559,24 @@ async def handle_request_with_rejection(
             )
             return
 
-        with self._wrap_user_method_call(request_metadata, request_args):
-            yield ReplicaQueueLengthInfo(
-                accepted=True,
-                # NOTE(edoakes): `_wrap_user_method_call` will increment the number
-                # of ongoing requests to include this one, so re-fetch the value.
-                num_ongoing_requests=self.get_num_ongoing_requests(),
+        with self._wrap_user_method_call(
+            request_metadata, request_args
+        ) as status_code_callback:
+            yield pickle.dumps(
+                ReplicaQueueLengthInfo(
+                    accepted=True,
+                    # NOTE(edoakes): `_wrap_user_method_call` will increment the number
+                    # of ongoing requests to include this one, so re-fetch the value.
+                    num_ongoing_requests=self.get_num_ongoing_requests(),
+                )
             )
```

Comment:
```suggestion
            yield ReplicaQueueLengthInfo(
                accepted=True,
                # NOTE(edoakes): `_wrap_user_method_call` will increment the number
                # of ongoing requests to include this one, so re-fetch the value.
                num_ongoing_requests=self.get_num_ongoing_requests(),
            )
```
I think with the new interface this is pickled in `ReplicaActor`

Reply:
oh good call!